### DISCUSSION TITLE
How does the action write back to the repo?

### DISCUSSION THREAD

**Discussion by mxcl** [githubocto/flat-action#44] [Q&A] [answered]
Pushing/committing to the repo from a GitHub Action generally requires a PAT. How does this action do it without one?

**Comment by mxcl** [comment, marked as answer]
Forget it, I was confused. The `GITHUB_TOKEN` provided by GHA *can* write back to the repo.

**Reply by bxb100** [reply]
exactly, the checkout action do this stuff, but i dont know why this repo can using `flat-data` account
